### PR TITLE
Updated copy and added trans tags where spotted missing

### PR DESCRIPTION
**Pull request by CSergienko**
@phala @chrislombaard Please have a look gents.

### CODE REVIEW

File: nurseconnect/templates/core/home_page_before_login.html [hunk 1]
```diff
@@ -2,18 +2,17 @@
 
 <form action="" class="Splash Splash--bgNurse">
     <div class="Splash-body">
-        <h1 class="Splash-headline">Join our community. <br/>We're here to support you.</h1>
-        <p class="Splash-strapline">NurseConnect will send valuable weekly SMS messages about maternal and newborn
-            health, family planning, MomConnect tips, service feedback, and motivational encouragement.</p>
+        <h1 class="Splash-headline">{% trans 'Welcome to our community of learning and support!' %}</h1>
```
Comment (chrislombaard) [inline nurseconnect/templates/core/home_page_before_login.html]: Please use double quotes

File: nurseconnect/templates/profiles/register.html [hunk 2]
```diff
@@ -52,18 +57,17 @@ <h1 class="Splash-headline">Welcome.<br> Let's get you signed up.</h1>
...
                         <li class="ButtonGroup-item">
-                            <input class="Button" type="submit" value="{% trans "Register" %}"/>
+                            <input class="Button" type="submit" value="{% trans "Sign Up" %}"/>
                         </li>
                         <input type="hidden" name="next" value="{% url "home" %}"/>
                         <li class="ButtonGroup-item">
                             <div class="Message Message--subtle Message--centered">
                                 <p class="Message-caption">
-                                    Already have an account? Login <a href="{% url "auth_login" %}">here</a>
+                                    Already have an account? Log in <a href="{% url "auth_login" %}">here</a>
```
Comment (markupguy) [inline nurseconnect/templates/profiles/register.html]: @CSergienko Missing trans tags :)

File: nurseconnect/templates/core/home_page_before_login.html [hunk 3]
```diff
@@ -2,18 +2,17 @@
...
+        <p class="Splash-strapline">{% trans 'Join our daily discussion topic and stay updated on maternal and newborn care. Our workplace tips and self-care info will motivate and encourage you!' %}</p>
```
Comment (chrislombaard) [inline nurseconnect/templates/core/home_page_before_login.html]: Double quotes.

File: nurseconnect/templates/core/home_page_before_login.html [hunk 4]
```diff
@@ -2,18 +2,17 @@
...
     </div>
     <div class="Splash-actions">
         <ul class="ButtonGroup">
             <li class="ButtonGroup-item">
-                <a href="{% url "user_register" %}" class="Button">Sign up now</a>
+                <a href="{% url "user_register" %}" class="Button">{% trans 'Sign up now' %}</a>
```
Comment (chrislombaard) [inline nurseconnect/templates/core/home_page_before_login.html]: Double quotes

**Review by markupguy — COMMENTED** [review]
+1, just the one missing trans tag. Thanks dude!

File: nurseconnect/templates/core/home_page_before_login.html [hunk 5]
```diff
@@ -2,18 +2,17 @@
...
             </li>
             <li class="ButtonGroup-item">
-                <a href="{% url "auth_login" %}" class="Button Button--inverted">Login</a>
+                <a href="{% url "auth_login" %}" class="Button Button--inverted">{% trans 'Log In' %}</a>
```
Comment (chrislombaard) [inline nurseconnect/templates/core/home_page_before_login.html]: DQ

File: nurseconnect/templates/profiles/editprofile.html [hunk 6]
```diff
@@ -5,7 +5,8 @@
     <form method="POST" action="{% url 'edit_my_profile' 'edit-settings' %}" class="Form Form--padded">
         <fieldset class="Form-fieldGroup">
             <a href="{% url "edit_my_profile" "edit-settings" %}" class="Form-groupStateToggle">{% trans "Edit" %}</a>
-            <legend class="Form-groupHeading">Settings</legend>
+            <legend class="Form-groupHeading">{% trans "Personal Info" %}</legend>
+            <p class="Form-message">To edit your name, surname or cellphone number, click on the Edit button. Then, click on ‘Update Profile’. Next time you log in, use your new cellphone number. </p>
```
Comment (chrislombaard) [inline nurseconnect/templates/profiles/editprofile.html]: Add a trans tag to text with double quotes

File: nurseconnect/templates/profiles/register.html [hunk 7]
```diff
@@ -22,15 +22,20 @@ <h1 class="Splash-headline">Welcome.<br> Let's get you signed up.</h1>
                         <label for="pword" class="Form-fieldLabel">{{ form.password.label }}</label>
                         {{ form.password }}
                         {{ form.password.errors }}
+                        <div class="Form-message">
+                            {% trans 'Tip: A good password has 4 characters and a mix of capitals and numbers. ' %}
```
Comment (chrislombaard) [inline nurseconnect/templates/profiles/register.html]: DQ

File: nurseconnect/templates/profiles/register.html [hunk 8]
```diff
@@ -22,15 +22,20 @@ <h1 class="Splash-headline">Welcome.<br> Let's get you signed up.</h1>
...
                     <div class="Form-field Form-field--required">
                         <label for="pwordconf" class="Form-fieldLabel">{{ form.confirm_password.label }}</label>
                         {{ form.confirm_password }}
                         {{ form.confirm_password.errors }}
                     </div>
                 </fieldset>
+                <p class="Form-message Form-message--centered"><i class="Form-requiredExample">*</i> {% trans "required field" %}</p>
                 <fieldset class="Form-fieldGroup">
                     <legend class="Form-groupHeading">Security questions</legend>
+                    <p>{% trans 'These will help you log in later if you forget your password.' %}</p>
```
Comment (chrislombaard) [inline nurseconnect/templates/profiles/register.html]: DQ

File: nurseconnect/templates/profiles/register.html (hunk 2 above)
Comment (chrislombaard) [inline nurseconnect/templates/profiles/register.html]: Please add a trans tag with double quotes.

**Review by chrislombaard — CHANGES_REQUESTED** [review]
Just those quotes and trans tags please.

**Review by chrislombaard — APPROVED** [review]

**Comment by chrislombaard** [comment]
+1

**Pull request merged**